Dockerfile

```diff
@@ -99,7 +105,8 @@ RUN export PATH="/opt/miniconda-latest/bin:$PATH" \
     && sync && conda clean -tipsy && sync \
     && bash -c "source activate base \
     &&   pip install --no-cache-dir --editable \
-             '/src/heudiconv[all]'" \
+             '/src/heudiconv[all]' \
+	     '/src/bidsphysio'" \
```

Comment:
this file is generated, shouldn't be modified directly. there is `utils/gen-docker-image.sh` to be tuned but even that one souldn't be changed in this case ;)

please first upload biosphysio to pypi and then just add it to `"extras"` within `heudiconv/info.py` nearby datalad.